HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -408,6 +408,14 @@ def customiseFor17098(process):
        producer.copyTrajectories = cms.untracked.bool(False)
     return process
 
+# Configurable Pixel Cluster Shape in LowPtSeedComparitor
+def customiseFor17393(process):
+    for producer in producers_by_type(process,"PixelTripletHLTEDProducer"):
+         if hasattr(producer,'SeedComparitorPSet'):
+             if (producer.SeedComparitorPSet.ComponentName.value() == 'LowPtClusterShapeSeedComparitor'):
+                  producer.SeedComparitorPSet.clusterShapeHitFilter = cms.string('ClusterShapeHitFilter')
```

Comment:
@ebrondol , in this way you are overwriting clusterShapeHitFilter in all 'LowPtClusterShapeSeedComparitor' component.
In other word we cannot control the clusterShapeHitFilter value from the HLT configuration, because it will be always overwritten by 'ClusterShapeHitFilter'. Did you want this? Did you forget to add `if not hasattr(producer.SeedComparitorPSet,'clusterShapeHitFilter')` ? Thanks for the check, Silvio.